lib/cog/commands/relay_group.ex

```diff
@@ -50,7 +53,7 @@ defmodule Cog.Commands.RelayGroup do
         if Helpers.flag?(req.options, "help") do
           show_usage
         else
-          show_usage(error(:required_subcommand))
+          RelayGroup.List.list(req, args)
```

Comment:
I think I'm going to go back through all these subcommands and convert them all over to calling their main function `run` as a separate PR.